docs/sql-ref-functions-builtin.md

```diff
@@ -17,202 +17,125 @@ license: |
   limitations under the License.
 ---
 
-{% for static_file in site.static_files %}
```

Comment:
Actually it was my bad. we still need this if-else because of `SKIP_API=1 bundle exec jekyll build`. It fails as below

```
Configuration file: /.../workspace/forked/spark/docs/_config.yml
            Source: /.../workspace/forked/spark/docs
       Destination: /.../workspace/forked/spark/docs/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
  Liquid Exception: Could not locate the included file '_generated_function_html/agg-funcs-table.html' in any of ["/.../workspace/forked/spark/docs/"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in sql-ref-functions-builtin.md
...
```

Reply:
Let me revert this for now (as it technically breaks the documentation build)

Reply:
Hmm, that's strange. Does the PR build not build documentation? Because the build was passing as of a9ae743eb66004dce9a9dd7e7ca24ef17823d5aa (the latest commit in the PR), and the merge into master at e4b5977c9b7b808d32a6370dccc33eaeb235085e also had a passing build.

Here's the documentation build after the merge: https://github.com/apache/spark/actions/runs/7294850973/job/19880408450

What am I missing?

Reply:
I think we can just revive the `if`s there, and I think we can just go ahead and merge. Let me reopen this PR.

Reply:
Oops, the branch was removed so I can't reopen the PR 😢

Reply:
No worries. I will reopen the PR.

I will also tweak things, if possible, so that `SKIP_API=1` will work even if the SQL API docs have never been generated before, _without_ requiring all of the conditional includes, because they really clutter things up.

Reply:
I've opened #44627 to address the build issue so we are not caught off guard like this again.

Reply:
Yeah .. as long as those options work (e.g., `SKIP_API=1.`), it should be fine.